### PR TITLE
Desktop version of Portfolio

### PR DESCRIPTION
**Pull request by Solobarine**
This is a pull request to review the desktop version of my Portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+        <div id="frames">
+          <div id="recent-img"><img src="./image/Img_placeholder.png" alt=""></div>
+          <h3 class="stories">Multi-Post Stories</h3>
+          <p class="summary">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="bar">
+            <span>CSS</span>
+            <span>HTML</span>
+            <span>Bootstrap</span>
+            <span>Ruby</span>
```
Comment (ishimwezachee) [inline index.html:56]: - [ ]  working on the desktop version should not affect your mobile version. I can see that working on the desktop version has affected you mobile design, please fix it. 
Your design  |  Figma design 
image  |  image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    <section id="opening">
+      <div id="intro_word">
+      <h1 id="title">Hey There. I’m Solomon</h1>
+      <h2 id="job">I’m a Software Developer</h2>
+      <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+        </div>
+      <ul id="social">
+        <li><a href="https://github.com/Solobarine"><i class="social-icons fa-brands "></i><img src="./image/social_media/github.png" alt="my github handle" width="20" height="20"></a></li>
+        <li><a href="https://www.linkedin.com/in/solomnakpuru/"><i class="social-icons fa-brands "></i><img src="./image/social_media/linkedin.png" alt="my linkedin handle" width="20" height="20" ></a></li>
```
Comment (ishimwezachee) [inline index.html]: - [ ]  Consider following the design from Figma. You have different spaces size compare to the one provided from the Figma design.
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+      <h2 id="work_title">My Recent Works</h2>
+
+      <div id="projects">
+
+        <div id="frames">
+          <div id="recent-img"><img src="./image/Img_placeholder.png" alt=""></div>
+          <h3 class="stories">Multi-Post Stories</h3>
+          <p class="summary">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="bar">
+            <span>CSS</span>
```
Comment (ishimwezachee) [inline index.html:53]: - [ ]  Your project is not responsive as it supposed to. its still show projects as if its in mobile version , I guess flex box will help you to make them spread the the whole page. 

Your Design

image  

Figma design

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        <ul id="skills">
+          <li class="ability-title"><span class="bullet" id="skills-bullet"></span>Skills</li>
+          <li>Codekit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
```
Comment (ishimwezachee) [inline index.html:172]: - [ ]  Your about me section should be similar to the one from figma design, make it responsive as well, and make sure that it doesn't affect your mobile version  design 

Your design 
image

Figma design 

image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Solobarine , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Linters are passing 👍 
Project is running successfully 👍 
PR is well described 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Solobarine** [comment]
This is the last pull request I have

**Review by sumairq — APPROVED** [review]
### Hi @Solobarine  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions
- [Optional] Kindly fix the path to images , as the background images are not displaying, otherwise your project is good to go. Cheers 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Solobarine** [comment]
Thank you very much Sumairq

**Pull request merged**